### ISSUE TITLE
[Usability] User experiences an unexpected interaction with a component.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#78950]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** Identity Core Team
**Product:** Terms of Use (ToU)
**Feature:** Terms of Use (ToU)

## Findings details
**VA.gov Experience Standard - issue:** User experiences an unexpected interaction with a component.
**VA.gov Experience Standard - category:** Usability
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

When you're not authenticated, there is a "Sign in to VA.gov" link near the top of the page. Users (especially assistive technology users) will expect a link to be navigational and take them somewhere else. Instead, clicking the link opens a modal on the same page.

## Recommended action
In most (all?) other instances of logging in on VA.gov we use a `button` rather than a link. This is a little bit of a funny edge case, since all of these buttons and the sign in link used here do result in a URL change in the browser address bar --- but the behavior that a user experiences is just opening a modal, which is solidly non-navigational. Unless there's a strong reason for deviating from this pattern, the "Sign in to VA.gov" element should be a link.

## References

- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by allison0034** [comment]
Hi @asg5704 Brian is out and I think I can help.

We do have a [defined pattern if the sign in opens a modal](https://design.va.gov/templates/forms/introduction#sign-in) that both Design System team and CAIA were involved with. Unfortunately some pages that have a link instead of a button are outdated or we just missed, but the correct component would be a button. Does this help?

**Comment by allison0034** [comment]
@asg5704 it is always great to reevaluate things!

Take a look at [Storybook](https://design.va.gov/storybook/?path=/docs/uswds-va-modal--default) and you can see the code.

If you have more questions feel free to post in the platform-design-system Slack channel. I can't remember the exact reason for the decision for "Create an account..." to be links. Everything else I have ever seen in a modal is a button because it closes the modal.
